src/interceptors/fetch.js

```diff
@@ -4,6 +4,13 @@ import parseUrl from 'parse-url';
 
 const nativeFetch = window.fetch;
 
+//TODO: Handle response headers
+let fakeResponse = function(response = {}) {
+  const responseStr = JSON.stringify(response);
+
+  return new Response(responseStr);
```

Comment:
I think Response can be constructed using any object? Do we have to stringify it first?